src/main/resources/application.yaml

```diff
@@ -16,7 +19,7 @@ kafka:
       name: slcb
 
 application:
-  bucket-name: paymenthub-ee-dev
+  bucket-name: paymenthub-ee
 
```

Comment:
This change was already added, you may have to rebase

Reply:
@fynmanoj I see that bucket-name is still `paymenthub-ee-dev` in  master branch. Could you please confirm if the PR is merged?